src/data/schedule.ts

```diff
@@ -0,0 +1,65 @@
+import { HomeAssistant } from "../types";
+
+export const weekdays = [
+  "sunday",
+  "monday",
+  "tuesday",
+  "wednesday",
+  "thursday",
+  "friday",
+  "saturday",
+];
+
+export interface ScheduleDay {
+  from: string;
+  to: string;
+}
+
+export interface Schedule {
+  id: string;
+  name: string;
+  icon?: string;
+  monday?: ScheduleDay[];
+  tuesday?: ScheduleDay[];
+  wednesday?: ScheduleDay[];
+  thursday?: ScheduleDay[];
+  friday?: ScheduleDay[];
+  saturday?: ScheduleDay[];
+  sunday?: ScheduleDay[];
```

Comment:
```suggestion
  [day: typeof weekdays[number]]?: ScheduleDay[];
```

Reply:
Not sure if this is better... just more confusing to read later maybe haha

I added it with the correct syntax